sdcm/utils/gce_utils.py

```diff
@@ -30,11 +30,14 @@
 
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
+    assert region.startswith("us-east1") or region.startswith("us-west1"), "only supports two regions"
```

Comment:
I think it's better to prepare from now to the option we will have list of supported regions not just 2.

Reply:
The only thing we need to do for support more gce region is creating sct builder in the region by an image:
- https://console.cloud.google.com/compute/machineImages/details/sct-builder-image-20210716-v1?project=skilled-adapter-452

I will add
GCE_SUPPORTED_REGIONS = ["us-east1", "us-west1"]